pandas/io/excel/_openpyxl.py

```diff
@@ -93,7 +62,7 @@ def _convert_to_style_kwargs(cls, style_dict):
         """
         _style_key_map = {"borders": "border"}
 
-        style_kwargs = {}
+        style_kwargs: Dict[str, Any] = {}
```

Comment:
Style object has type Serialisable. We could import it instead of using Any. But mypy resolves it as Any on my machine.

Reply:
Any is harder to track, so an import maybe best here